drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -169,14 +171,30 @@ int nrf_wifi_if_send(const struct device *dev,
 		goto out;
 	}
 
-	if ((vif_ctx_zep->if_carr_state != NRF_WIFI_FMAC_IF_CARR_STATE_ON) ||
-	    (!vif_ctx_zep->authorized && !is_eapol(pkt))) {
-		goto out;
-	}
+#ifdef CONFIG_NRF700X_RAWDATA_TX
```

Comment:
In Zephyr the philosophy of defines is that `if (IS_ENABLED(CONFIG_X)))` the main rationale being the code will be compiled tested.

Reply:
okay - will make the change. Used the same method as used for other features